raiden/blockchain/abi.py

```diff
@@ -230,6 +251,20 @@ def instantiate(self):
             )
 
             self.is_instantiated = True
+            self.init_contract_versions()
+
+    def init_contract_versions(self):
+        contracts = [('HumanStandardToken.sol', CONTRACT_HUMAN_STANDARD_TOKEN),
```

Comment:
style:

new line after opening the array and new line before closing the array